package.json

```diff
@@ -35,7 +35,7 @@
   "scripts": {
     "prepare": "husky install",
     "env:prep": "npm run build:lib && npm link && npm link @maplibre/maplibre-gl-directions",
-    "dev:lib": "npm run check:lib && tsc --project ./tsconfig.lib.json && vite build --watch --mode development --config vite.lib.config.ts",
```

Comment:
Why did you remove the `npm run check:lib` part?

Reply:
Running check once before start of dev server has little sense cause check is not watching for changes. I added tsc:lib:watch instead since it's needed to regenerate typings on code changes for correct types to be discovered in the demo app during development. I simply ran it in a separate terminal tab but it might be called from the dev:lib with run-p to start it in parallel with dev server if you deem necessary.

Reply:
Obviously it gets run only once, when starting the dev server. As far as I know, there's no way to run in continuosly (sort of a watch-mode). But running it once is anyway better than not running it a all. I think this check shouldn't be removed.

Reply:
But what's the point in checking before start? It's called during build, this should be enough.

Reply:
BTW watch option has been added to `svelte-check` https://github.com/sveltejs/language-tools/issues/287

Reply:
So, taking the newly-added `watch` mode into account, I have two options for you:
1. Either configure the script properly to run both the server and the checker alongside each other (in parallel?)
2. Or restore the removed one-time check

I personally use it quite often with just restarting the dev server. At least because it's way faster than building the whole package just to check the Svelte correctness.

That's the last requirement before we're good to go with this PR.